notary-commons/src/main/kotlin/sidechain/ChainListener.kt

```diff
@@ -11,11 +11,11 @@ interface ChainListener<Block> : Closeable {
     /**
      * @return Observable on blocks that committed to the network
      */
-    fun getBlockObservable(): Result<io.reactivex.Observable<Block>, Exception>
+    fun getBlockObservable(autoAck : Boolean = true): Result<io.reactivex.Observable<Block>, Exception>
```

Comment:
`false` value is never used. Should the parameter be removed?

Reply:
This is a very important thing, to use noAutoAck we need to add something else, let it be true for now